Sources/Task.swift

```diff
@@ -504,20 +504,22 @@ extension Task {
 							if terminationStatus == EXIT_SUCCESS {
 								// Wait for stderr to finish, then pass
 								// through stdout.
-								disposable += stderrAggregated
+								let disposable = stderrAggregated
```

Comment:
We can continue to use `+=` operator 😃 

https://github.com/ReactiveCocoa/ReactiveSwift/pull/482